int16 epoch
int16 episode
float32 prev_score
float32 max_score
float32 act_score
bool training
